### ISSUE TITLE
Homebridge startup exists without error after initializing the z-way devices

### ISSUE THREAD

**Issue by eicosan** [SphtKr/homebridge-zway#21]
Hi all,

when starting up homebridge, the process initializes the z-way devices but then quits without error message. 

Debug Log:

```
root@raspberrypi:/tmp#  DEBUG=* homebridge -U ~/.homebridge/
*** WARNING *** The program 'nodejs' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=nodejs>
*** WARNING *** The program 'nodejs' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=nodejs&f=DNSServiceRegister>
[2016-03-17 14:01:09] Loaded plugin: homebridge-zway
[2016-03-17 14:01:09] Registering accessory 'homebridge-zway.ZWayServer'
[2016-03-17 14:01:09] Registering platform 'homebridge-zway.ZWayServer'
[2016-03-17 14:01:09] ---
[2016-03-17 14:01:09] Loaded config.json with 0 accessories and 1 platforms.
[2016-03-17 14:01:09] ---
[2016-03-17 14:01:09] Loading 1 platforms...
[2016-03-17 14:01:09] Initializing ZWayServer platform...
  ZWayServer Fetching Z-Way devices... +0ms
  ZWayServer Authenticating... +255ms
  ZWayServer Authenticated. Resubmitting original request... +111ms
  ZWayServer Got typeKey battery.Battery for vdev BatteryPolling_8 +66ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Got typeKey sensorMultiline.Temperature for vdev OpenWeather_20 +125ms
  ZWayServer { deviceType: 'sensorMultiline', probeTitle: 'Temperature', probeType: '' } +0ms
  ZWayServer Got typeKey toggleButton for vdev LightScene_13 +3ms
  ZWayServer { deviceType: 'toggleButton', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Got typeKey toggleButton for vdev LightScene_14 +2ms
  ZWayServer { deviceType: 'toggleButton', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey buttonControl for vdev ZWayVDev_zway_Remote_2-1-1-B +2ms
  ZWayServer { deviceType: 'buttonControl', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey buttonControl for vdev ZWayVDev_zway_Remote_2-124-1-B +2ms
  ZWayServer { deviceType: 'buttonControl', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +3ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +1ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +2ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Got typeKey sensorBinary.alarmSensor_general_purpose for vdev ZWayVDev_zway_2-0-156-0-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarmSensor_general_purpose' } +0ms
  ZWayServer Got typeKey switchBinary for vdev ZWayVDev_zway_3-0-37 +2ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_3-0-49-4 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_3-0-50-0 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_kilowatt_per_hour' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_3-0-50-2 +3ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_watt' } +0ms
  ZWayServer Got typeKey switchBinary for vdev Sonos_Device_Play_192.168.2.50_22 +2ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-0-49-4 +8ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-0 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_kilowatt_per_hour' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-2 +6ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_watt' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-2-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-4-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-5-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-6-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-8-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-9-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-1-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-2-38 +3ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-1-49-4 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +1ms
  ZWayServer Got grouped device BatteryPolling_8 consiting of devices: +1ms
  ZWayServer BatteryPolling_8 - battery.Battery +1ms
  ZWayServer WARN: Didn't find suitable device class! +0ms
  ZWayServer Got grouped device OpenWeather_20 consiting of devices: +1ms
  ZWayServer OpenWeather_20 - sensorMultiline.Temperature +0ms
  ZWayServer WARN: Didn't find suitable device class! +0ms
  ZWayServer Got grouped device LightScene_13 consiting of devices: +0ms
  ZWayServer LightScene_13 - toggleButton +1ms
  ZWayServer WARN: Didn't find suitable device class! +0ms
  ZWayServer Got grouped device LightScene_14 consiting of devices: +0ms
  ZWayServer LightScene_14 - toggleButton +1ms
  ZWayServer WARN: Didn't find suitable device class! +0ms
  ZWayServer Got grouped device ZWayVDev_zway_Remote_2-1-1-B consiting of devices: +0ms
  ZWayServer ZWayVDev_zway_Remote_2-1-1-B - buttonControl +0ms
  ZWayServer WARN: Didn't find suitable device class! +1ms
  ZWayServer Got grouped device ZWayVDev_zway_Remote_2-124-1-B consiting of devices: +0ms
  ZWayServer ZWayVDev_zway_Remote_2-124-1-B - buttonControl +0ms
  ZWayServer WARN: Didn't find suitable device class! +0ms
  ZWayServer Got grouped device ZWayVDev_2-0 consiting of devices: +1ms
  ZWayServer ZWayVDev_zway_2-0-48-1 - sensorBinary.General purpose +0ms
  ZWayServer ZWayVDev_zway_2-0-49-1 - sensorMultilevel.Temperature +0ms
  ZWayServer ZWayVDev_zway_2-0-49-3 - sensorMultilevel.Luminiscence +0ms
  ZWayServer ZWayVDev_zway_2-0-128 - battery.Battery +1ms
  ZWayServer ZWayVDev_zway_2-0-156-0-A - sensorBinary +0ms
  ZWayServer Got grouped device ZWayVDev_3-0 consiting of devices: +1ms
  ZWayServer ZWayVDev_zway_3-0-37 - switchBinary +0ms
  ZWayServer ZWayVDev_zway_3-0-49-4 - sensorMultilevel.Power +1ms
  ZWayServer ZWayVDev_zway_3-0-50-0 - sensorMultilevel.Electric +0ms
  ZWayServer ZWayVDev_zway_3-0-50-2 - sensorMultilevel.Electric +0ms
  ZWayServer Got grouped device Sonos_Device_Play_192.168.2.50_22 consiting of devices: +1ms
  ZWayServer Sonos_Device_Play_192.168.2.50_22 - switchBinary +1ms
  ZWayServer Got grouped device Sonos_Device_Volume_192.168.2.50_22 consiting of devices: +0ms
  ZWayServer Sonos_Device_Volume_192.168.2.50_22 - switchMultilevel +1ms
  ZWayServer Got grouped device ZWayVDev_4-0 consiting of devices: +0ms
  ZWayServer ZWayVDev_zway_4-0-49-4 - sensorMultilevel.Power +0ms
  ZWayServer ZWayVDev_zway_4-0-50-0 - sensorMultilevel.Electric +0ms
  ZWayServer ZWayVDev_zway_4-0-50-2 - sensorMultilevel.Electric +1ms
  ZWayServer ZWayVDev_zway_4-0-113-4-2-A - sensorBinary +1ms
  ZWayServer ZWayVDev_zway_4-0-113-8-4-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-8-5-A - sensorBinary +1ms
  ZWayServer ZWayVDev_zway_4-0-113-8-6-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-8-8-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-8-9-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-9-1-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-38 - switchMultilevel +1ms
  ZWayServer ZWayVDev_zway_4-0-113-4-254-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-8-254-A - sensorBinary +0ms
  ZWayServer ZWayVDev_zway_4-0-113-9-254-A - sensorBinary +0ms
  ZWayServer Got grouped device ZWayVDev_4-1 consiting of devices: +1ms
  ZWayServer ZWayVDev_zway_4-1-38 - switchMultilevel +2ms
  ZWayServer ZWayVDev_zway_4-1-49-4 - sensorMultilevel.Power +4ms
  ZWayServer Got grouped device ZWayVDev_4-2 consiting of devices: +4ms
  ZWayServer ZWayVDev_zway_4-2-38 - switchMultilevel +0ms
[2016-03-17 14:01:10] Initializing platform accessory 'Motion Sensor - Temperatur'...
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +22ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +0ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +55ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +0ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +3ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +5ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +0ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +3ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +1ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +4ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +1ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +5ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +1ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +2ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Got typeKey sensorBinary.alarmSensor_general_purpose for vdev ZWayVDev_zway_2-0-156-0-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarmSensor_general_purpose' } +0ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +1ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +1ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +6ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +6ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +1ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +2ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +5ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +1ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +5ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +0ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +1ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +1ms
  ZWayServer Got typeKey sensorBinary.alarmSensor_general_purpose for vdev ZWayVDev_zway_2-0-156-0-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarmSensor_general_purpose' } +0ms
  ZWayServer Got typeKey sensorBinary.General purpose for vdev ZWayVDev_zway_2-0-48-1 +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: 'General purpose', probeType: 'general_purpose' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Temperature for vdev ZWayVDev_zway_2-0-49-1 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Temperature', probeType: 'temperature' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Luminiscence for vdev ZWayVDev_zway_2-0-49-3 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Luminiscence', probeType: 'luminosity' } +0ms
  ZWayServer Got typeKey battery.Battery for vdev ZWayVDev_zway_2-0-128 +0ms
  ZWayServer { deviceType: 'battery', probeTitle: 'Battery', probeType: '' } +0ms
  ZWayServer Loaded services for Motion Sensor - Temperatur +1ms
[2016-03-17 14:01:10] Initializing platform accessory 'Steckdose - Schalter'...
  ZWayServer Got typeKey switchBinary for vdev ZWayVDev_zway_3-0-37 +25ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Got typeKey switchBinary for vdev ZWayVDev_zway_3-0-37 +13ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Loaded services for Steckdose - Schalter +6ms
[2016-03-17 14:01:10] Initializing platform accessory 'Sonos Büro'...
  ZWayServer Got typeKey switchBinary for vdev Sonos_Device_Play_192.168.2.50_22 +11ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Got typeKey switchBinary for vdev Sonos_Device_Play_192.168.2.50_22 +4ms
  ZWayServer { deviceType: 'switchBinary', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Loaded services for Sonos Büro +3ms
[2016-03-17 14:01:10] Initializing platform accessory 'Sonos Büro - Lautstärke'...
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +8ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +5ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +4ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev Sonos_Device_Volume_192.168.2.50_22 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: '' } +1ms
  ZWayServer Loaded services for Sonos Büro - Lautstärke +1ms
[2016-03-17 14:01:10] Initializing platform accessory 'Deckenlicht Büro'...
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +8ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +3ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-0-49-4 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-0 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_kilowatt_per_hour' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-2 +2ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_watt' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-2-A +5ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-4-A +3ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-5-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-6-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-8-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-9-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-1-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +1ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-0-49-4 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-0 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_kilowatt_per_hour' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Electric for vdev ZWayVDev_zway_4-0-50-2 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Electric', probeType: 'meterElectric_watt' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-2-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-4-A +5ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-5-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-6-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-8-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-9-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-1-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-0-38 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey sensorBinary.alarm_heat for vdev ZWayVDev_zway_4-0-113-4-254-A +4ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_heat' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_power for vdev ZWayVDev_zway_4-0-113-8-254-A +2ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_power' } +1ms
  ZWayServer Got typeKey sensorBinary.alarm_system for vdev ZWayVDev_zway_4-0-113-9-254-A +1ms
  ZWayServer { deviceType: 'sensorBinary', probeTitle: undefined, probeType: 'alarm_system' } +0ms
  ZWayServer Loaded services for Deckenlicht Büro +1ms
[2016-03-17 14:01:10] Initializing platform accessory 'Deckenlicht Büro'...
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +8ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +5ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +4ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +2ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +0ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-1-49-4 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Got typeKey switchMultilevel for vdev ZWayVDev_zway_4-1-38 +1ms
  ZWayServer { deviceType: 'switchMultilevel', probeTitle: undefined, probeType: 'dimmer' } +1ms
  ZWayServer Got typeKey sensorMultilevel.Power for vdev ZWayVDev_zway_4-1-49-4 +1ms
  ZWayServer { deviceType: 'sensorMultilevel', probeTitle: 'Power', probeType: 'energy' } +0ms
  ZWayServer Loaded services for Deckenlicht Büro +1ms
root@raspberrypi:/tmp# 
```

Thanks,
Johannes

**Comment by SphtKr** [comment]
Hello,

Can you provide me a copy of your `devices` output? You'll need to login to the Z-Way standard UI--e.g. at `http://your.pi.i.p:8083/`, and then from that same browser window hit `http://your.pi.i.p:8083/ZAutomation/api/v1/devices` (replacing your.pi.i.p with your device's IP address of course), and attach or post the contents here.

**Comment by eicosan** [comment]
Hi,

this is the requested output:

`{"data":{"structureChanged":true,"updateTime":1458284628,"devices":[{"creationTime":1453473440,"creatorId":8,"deviceType":"battery","h":-592588977,"hasHistory":false,"id":"BatteryPolling_8","location":0,"metrics":{"probeTitle":"Battery","scaleTitle":"%","title":"Battery digest 8","level":100},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458062903},{"creationTime":1458061306,"creatorId":20,"deviceType":"sensorMultiline","h":-1611286413,"hasHistory":false,"id":"OpenWeather_20","location":1,"metrics":{"scaleTitle":"°C","title":"München","probeTitle":"Temperature","zwaveOpenWeather":{"main":{"temp":283.55,"pressure":1016,"humidity":100,"temp_min":282.04,"temp_max":284.15},"weather":[{"id":701,"main":"Mist","description":"mist","icon":"50d"}],"wind":{"speed":3.1,"deg":40}},"level":10.4,"icon":"http://openweathermap.org/img/w/50d.png","country":"DE","flag":"http://openweathermap.org/images/flags/de.png","multilineType":"openWeather"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458061306},{"creationTime":1453473427,"creatorId":13,"deviceType":"toggleButton","h":557861323,"hasHistory":true,"id":"LightScene_13","location":1,"metrics":{"level":"on","icon":"","title":"Light Scene 13"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458117215},{"creationTime":1453473427,"creatorId":14,"deviceType":"toggleButton","h":557861324,"hasHistory":true,"id":"LightScene_14","location":1,"metrics":{"level":"on","icon":"","title":"Light Scene 14"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458118415},{"creationTime":1453473438,"deviceType":"buttonControl","h":434271221,"hasHistory":true,"id":"ZWayVDev_zway_Remote_2-1-1-B","location":1,"metrics":{"icon":"","title":"Button Fibar Group 1-1","level":"on","change":""},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458284584},{"creationTime":1453473438,"deviceType":"buttonControl","h":868717111,"hasHistory":true,"id":"ZWayVDev_zway_Remote_2-124-1-B","location":0,"metrics":{"icon":"","title":"Button Fibar Group 124-1","level":"off","change":""},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458061500},{"creationTime":1453473440,"creatorId":1,"deviceType":"sensorBinary","h":1303282175,"hasHistory":true,"id":"ZWayVDev_zway_2-0-48-1","location":1,"metrics":{"probeTitle":"General purpose","scaleTitle":"","icon":"motion","level":"on","title":"Motion Sensor - Bewegung"},"permanently_hidden":false,"probeType":"general_purpose","tags":[],"visibility":true,"updateTime":1458284584},{"creationTime":1453473440,"creatorId":1,"deviceType":"sensorMultilevel","h":1303283136,"hasHistory":true,"id":"ZWayVDev_zway_2-0-49-1","location":1,"metrics":{"probeTitle":"Temperature","scaleTitle":"°C","level":23.2,"icon":"temperature","title":"Motion Sensor - Temperatur"},"permanently_hidden":false,"probeType":"temperature","tags":[],"visibility":true,"updateTime":1458284428},{"creationTime":1453473440,"creatorId":1,"deviceType":"sensorMultilevel","h":1303283138,"hasHistory":true,"id":"ZWayVDev_zway_2-0-49-3","location":1,"metrics":{"probeTitle":"Luminiscence","scaleTitle":"Lux","level":51,"icon":"luminosity","title":"Motion Sensor - Helligkeit"},"permanently_hidden":false,"probeType":"luminosity","tags":[],"visibility":true,"updateTime":1458061500},{"creationTime":1453473441,"creatorId":1,"deviceType":"battery","h":-927793024,"hasHistory":false,"id":"ZWayVDev_zway_2-0-128","location":0,"metrics":{"probeTitle":"Battery","scaleTitle":"%","level":100,"icon":"battery","title":"Battery 2-0 Fibar Group"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458062903},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorBinary","h":1129728498,"hasHistory":true,"id":"ZWayVDev_zway_2-0-156-0-A","location":1,"metrics":{"icon":"alarm","level":"off","title":"Motion Sensor - Alarm"},"permanently_hidden":false,"probeType":"alarmSensor_general_purpose","tags":[],"visibility":true,"updateTime":1458061500},{"creationTime":1453473441,"creatorId":1,"deviceType":"switchBinary","h":1107079066,"hasHistory":true,"id":"ZWayVDev_zway_3-0-37","location":1,"metrics":{"icon":"switch","title":"Steckdose - Schalter","level":"off"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458118415},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorMultilevel","h":-1248873822,"hasHistory":true,"id":"ZWayVDev_zway_3-0-49-4","location":1,"metrics":{"probeTitle":"Power","scaleTitle":"W","level":0,"icon":"energy","title":"Steckdose - Verbrauch Aktuell"},"permanently_hidden":false,"probeType":"energy","tags":[],"visibility":true,"updateTime":1458284402},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorMultilevel","h":-1248852684,"hasHistory":true,"id":"ZWayVDev_zway_3-0-50-0","location":1,"metrics":{"probeTitle":"Electric","scaleTitle":"kWh","level":8.08,"icon":"meter","title":"Steckdose - Verbrauch Summe"},"permanently_hidden":false,"probeType":"meterElectric_kilowatt_per_hour","tags":[],"visibility":true,"updateTime":1458284402},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorMultilevel","h":-1248852682,"hasHistory":true,"id":"ZWayVDev_zway_3-0-50-2","location":1,"metrics":{"probeTitle":"Electric","scaleTitle":"W","level":0,"icon":"meter","title":"Steckdose - Verbrauch ?"},"permanently_hidden":false,"probeType":"meterElectric_watt","tags":[],"visibility":true,"updateTime":1458283801},{"creationTime":1458060864,"creatorId":22,"deviceType":"switchBinary","h":1884175838,"hasHistory":true,"id":"Sonos_Device_Play_192.168.2.50_22","location":1,"metrics":{"title":"Sonos Büro","icon":"","level":"off"},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458282101},{"creationTime":1458060864,"creatorId":22,"deviceType":"switchMultilevel","h":-1360300604,"hasHistory":true,"id":"Sonos_Device_Volume_192.168.2.50_22","location":1,"metrics":{"title":"Sonos Büro - Lautstärke","icon":"","level":20},"permanently_hidden":false,"probeType":"","tags":[],"visibility":true,"updateTime":1458282101},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorMultilevel","h":493936513,"hasHistory":true,"id":"ZWayVDev_zway_4-0-49-4","location":"1","metrics":{"probeTitle":"Power","scaleTitle":"W","level":0,"icon":"energy","title":"Sensor Power 4-0-49-4 Fibar Group"},"permanently_hidden":false,"probeType":"energy","tags":[],"visibility":true,"updateTime":1458283801},{"creationTime":1453473441,"creatorId":1,"deviceType":"sensorMultilevel","h":493957651,"hasHistory":true,"id":"ZWayVDev_zway_4-0-50-0","location":"1","metrics":{"probeTitle":"Electric","scaleTitle":"kWh","level":4.2800004,"icon":"meter","title":"Meter Electric 4-0-50-0 Fibar Group"},"permanently_hidden":false,"probeType":"meterElectric_kilowatt_per_hour","tags":[],"visibility":true,"updateTime":1458283801},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorMultilevel","h":493957653,"hasHistory":true,"id":"ZWayVDev_zway_4-0-50-2","location":"1","metrics":{"probeTitle":"Electric","scaleTitle":"W","level":0,"icon":"meter","title":"Meter Electric 4-0-50-2 Fibar Group"},"permanently_hidden":false,"probeType":"meterElectric_watt","tags":[],"visibility":true,"updateTime":1458284402},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1710385310,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-4-2-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Heat 4-0-113-4-2 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_heat","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1714081316,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-4-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Power Management 4-0-113-8-4 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1714082277,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-5-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Power Management 4-0-113-8-5 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1714083238,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-6-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Power Management 4-0-113-8-6 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1714085160,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-8-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Power Management 4-0-113-8-8 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1714086121,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-9-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm Power Management 4-0-113-8-9 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorBinary","h":1715001954,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-9-1-A","location":"1","metrics":{"icon":"alarm","level":"off","title":"Alarm System 4-0-113-9-1 Fibar Group"},"permanently_hidden":false,"probeType":"alarm_system","tags":[],"visibility":true,"updateTime":1458063001},{"creationTime":1453473442,"creatorId":1,"deviceType":"switchMultilevel","h":1135708218,"hasHistory":true,"id":"ZWayVDev_zway_4-0-38","location":1,"metrics":{"icon":"multilevel","title":"Deckenlicht Büro","level":0},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458118416},{"creationTime":1453473442,"creatorId":1,"deviceType":"switchMultilevel","h":1135738009,"hasHistory":true,"id":"ZWayVDev_zway_4-1-38","location":0,"metrics":{"icon":"multilevel","title":"Deckenlicht Büro","level":0},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458248619},{"creationTime":1458062712,"creatorId":1,"deviceType":"switchMultilevel","h":1135767800,"hasHistory":true,"id":"ZWayVDev_zway_4-2-38","location":0,"metrics":{"icon":"multilevel","title":"Fibar Group Dimmer (4.2)","level":19},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458248240},{"creationTime":1453473442,"creatorId":1,"deviceType":"sensorMultilevel","h":522565664,"hasHistory":true,"id":"ZWayVDev_zway_4-1-49-4","location":0,"metrics":{"probeTitle":"Power","scaleTitle":"W","level":0,"icon":"energy","title":"Sensor Power 4-1-49-4 Fibar Group"},"permanently_hidden":false,"probeType":"energy","tags":[],"visibility":true,"updateTime":1458281189},{"creationTime":1453474298,"creatorId":1,"deviceType":"sensorBinary","h":-1291964163,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-4-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group Heat Alarm (4.0.113.4.254)"},"permanently_hidden":false,"probeType":"alarm_heat","tags":[],"visibility":true,"updateTime":1458284403},{"creationTime":1453474298,"creatorId":1,"deviceType":"sensorBinary","h":-2036916735,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-8-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group Power Management Alarm (4.0.113.8.254)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458284403},{"creationTime":1453474299,"creatorId":1,"deviceType":"sensorBinary","h":-1149413054,"hasHistory":true,"id":"ZWayVDev_zway_4-0-113-9-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group System Alarm (4.0.113.9.254)"},"permanently_hidden":false,"probeType":"alarm_system","tags":[],"visibility":true,"updateTime":1458284403},{"creationTime":1458240848,"creatorId":1,"deviceType":"sensorMultilevel","h":-2058220448,"hasHistory":true,"id":"ZWayVDev_zway_5-0-49-4","location":0,"metrics":{"probeTitle":"Power","scaleTitle":"W","level":0,"icon":"energy","title":"Fibar Group Power (5.0.49.4)"},"permanently_hidden":false,"probeType":"energy","tags":[],"visibility":true,"updateTime":1458283802},{"creationTime":1458240854,"creatorId":1,"deviceType":"sensorMultilevel","h":-2058199310,"hasHistory":true,"id":"ZWayVDev_zway_5-0-50-0","location":0,"metrics":{"probeTitle":"Electric","scaleTitle":"kWh","level":0,"icon":"meter","title":"Fibar Group Electric Meter (5.0.50.0)"},"permanently_hidden":false,"probeType":"meterElectric_kilowatt_per_hour","tags":[],"visibility":true,"updateTime":1458283202},{"creationTime":1458240854,"creatorId":1,"deviceType":"sensorMultilevel","h":-2058199308,"hasHistory":true,"id":"ZWayVDev_zway_5-0-50-2","location":0,"metrics":{"probeTitle":"Electric","scaleTitle":"W","level":0,"icon":"meter","title":"Fibar Group Electric Meter (5.0.50.2)"},"permanently_hidden":false,"probeType":"meterElectric_watt","tags":[],"visibility":true,"updateTime":1458284403},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1416982303,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-4-2-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Heat Alarm (5.0.113.4.2)"},"permanently_hidden":false,"probeType":"alarm_heat","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1420678309,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-4-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Power Management Alarm (5.0.113.8.4)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1420679270,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-5-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Power Management Alarm (5.0.113.8.5)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1420680231,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-6-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Power Management Alarm (5.0.113.8.6)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1420682153,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-8-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Power Management Alarm (5.0.113.8.8)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1420683114,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-9-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group Power Management Alarm (5.0.113.8.9)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240855,"creatorId":1,"deviceType":"sensorBinary","h":1421598947,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-9-1-A","location":0,"metrics":{"icon":"alarm","level":"off","title":"Fibar Group System Alarm (5.0.113.9.1)"},"permanently_hidden":false,"probeType":"alarm_system","tags":[],"visibility":true,"updateTime":1458240901},{"creationTime":1458240856,"creatorId":1,"deviceType":"switchMultilevel","h":1164337369,"hasHistory":true,"id":"ZWayVDev_zway_5-0-38","location":0,"metrics":{"icon":"multilevel","title":"Fibar Group Dimmer (5.0)","level":0},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458248658},{"creationTime":1458240864,"creatorId":1,"deviceType":"switchMultilevel","h":1164367160,"hasHistory":true,"id":"ZWayVDev_zway_5-1-38","location":0,"metrics":{"icon":"multilevel","title":"Fibar Group Dimmer (5.1)","level":0},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458248651},{"creationTime":1458240868,"creatorId":1,"deviceType":"switchMultilevel","h":1164396951,"hasHistory":true,"id":"ZWayVDev_zway_5-2-38","location":0,"metrics":{"icon":"multilevel","title":"Fibar Group Dimmer (5.2)","level":0},"permanently_hidden":false,"probeType":"dimmer","tags":[],"visibility":true,"updateTime":1458248646},{"creationTime":1458241804,"creatorId":1,"deviceType":"sensorBinary","h":215587646,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-4-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group Heat Alarm (5.0.113.4.254)"},"permanently_hidden":false,"probeType":"alarm_heat","tags":[],"visibility":true,"updateTime":1458284403},{"creationTime":1458241804,"creatorId":1,"deviceType":"sensorBinary","h":-529364926,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-8-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group Power Management Alarm (5.0.113.8.254)"},"permanently_hidden":false,"probeType":"alarm_power","tags":[],"visibility":true,"updateTime":1458283803},{"creationTime":1458241805,"creatorId":1,"deviceType":"sensorBinary","h":358138755,"hasHistory":true,"id":"ZWayVDev_zway_5-0-113-9-254-A","location":0,"metrics":{"icon":"alarm","level":"on","title":"Fibar Group System Alarm (5.0.113.9.254)"},"permanently_hidden":false,"probeType":"alarm_system","tags":[],"visibility":true,"updateTime":1458284404}]},"code":200,"message":"200 OK","error":null}`

**Comment by aivs** [comment]
I created 2 code devices (sensor binary, sensor multilevel) and 2 dummy devices (binary switch, multilevel switch).
In log I see that dummys created, ant then homebridge go out with out error.

**Comment by SphtKr** [comment]
@eicosan It appears your problem is happening because you have two accessories both named "Deckenlicht Büro", and apparently Homebridge requires all `displayName`s to be unique--and/or I need to do something so that duplicate names do not cause a problem in that case.

(I may also be swallowing error messages, I will have to look at that.)

I am marking this as a bug and will leave it open until the bug is fixed in this plugin, but in the meantime you can work around this by going into the Z-Way web interface and renaming one of your "Deckenlicht Büro" devices to a different name (just add a "2" or something)...that should let you finish launching. Check and see if you have any other duplicate names also, just in case.

@aivs I suppose it's possible that two of your devices also have the same name, but if not then your issue is probably a different one. Can you follow the instructions above to download your `devices` information and then post that in a second issue?

**Comment by eicosan** [comment]
Hi, 
the rename of the duplicate name resolved the startup of homebridge. 
But now I'm not able to connect to it from iOS. I already deleted it from settings menu and tried to re-add it, but the homebridge device is not found... Any idea?

Thanks!

**Comment by SphtKr** [comment]
@eicosan Sorry, just saw this. That depends. First, I think, try deleting the `persist` directory in your `~/.homebridge/`directory, and then try re-pairing in iOS. That should at least allow it to appear again to be paired. If it appears, but fails to pair, then there's some illegal output from the plugin(s) and we'll have to troubleshoot that.

**Comment by SphtKr** [comment]
Pushing this to 0.6.0 milestone for a few reasons. One, fixing this will cause all device UUIDs to change, which will force users to rebuild homekit scenes/groups/triggers/etc--this will probably happen when I implement the "plugin 2.0" API from Homebridge, so better to have this happen only once instead of twice. Two, going to the plugin 2.0 API may fix this by default.

Again, if anyone hits this same problem, the workaround is to make sure you don't have any Z-Way devices with duplicate names.

Depends #26

**Comment by klaasjoerg** [comment]
Hi together,
I have basically the same issue, but can't get it solved...:
Starting up homebridge suddenly simply exits right after Initializing some platform accessories. 

So first I deactivated all homebridge plugins, but still no luck. 
Next, as discussed above, I made sure, that I don't have any duplicate names (which I had indeed) and renamed those unqiuely. - still no luck.
I am running: homebridge-zway@0.5.0-alpha4 
Strange. Any ideas or recommendation where to look at / what to check?

**Comment by klaasjoerg** [comment]
Ok, too knowsie to wait. I continued deactivating devices in UI step by step. And suddenly, after deactivating most of them, it started to work. Will have to find out now, what devices is exactly causing this issue...   Let you know about it.
But, also one little recommendation:

Would be really a great help, if some "error messages" would be shown, if something goes wrong with homebridge-zway....

**Comment by SphtKr** [comment]
Actually if you run `DEBUG=ZWayServer homebridge` or DEBUG=* homebridge` you should get some more information. Possibly too much.

**Comment by klaasjoerg** [comment]
I found out, what's causing my mess here.
For some reason, (suddenly) the formerly working fibaro motion sensor is causing my trouble.
I have two of them and as soon as I activate the device "motion alarm" homebridge is quitting.
If I activate e.g. Fibar Group Luminiscence oder Temperature alarm, it's not causing any problems.

So it really seems to be the "Fibaro motion alarm" itself that's causing this.

**Comment by SphtKr** [comment]
@klaasjoerg Okay, (capture your `devices` data)[https://github.com/SphtKr/homebridge-zway#getting-zautomationapiv1devices] and probably open a new ticket, as this now looks like a separate issue. Also, see if you can figure out what version of Firmware you have on your Fibaro motion sensor. Some other folks are having trouble with that device (though I haven't heard of Homebridge quitting because of it) and the firmware version seems to make a difference. I have one myself but it causes me no problems, so there's probably something different in other firmware versions.

**Comment by SphtKr** [comment]
Oops, nope, I meant to leave this open to fix UUID conflicts for duplicate `name`s.
